### ISSUE TITLE
Add wasm-oidc-plugin

### ISSUE THREAD

**Issue by antonengelhardt** [istio-ecosystem/wasm-extensions#93]
Hey there,

i figured it might be useful to include https://github.com/antonengelhardt/wasm-oidc-plugin/ inside this repo. Or are only Wasm Extension written in C++ featured?

Cheers

**Comment by kyessenov** [comment]
No, it's any language, as long as it can be built and tested in CI.

**Comment by antonengelhardt** [comment]
@kyessenov Ok, thanks.

Should i open a PR? And how do you want it to be integrated?

**Comment by kyessenov** [comment]
Yes, you can open a PR. You might need to add a CI step for rust filters specifically, since the base probably doesn't have rust already.

**Comment by antonengelhardt** [comment]
@kyessenov Ok. Does it make sense to add it as a submodule? I have not worked with that before but maybe this makes updating it easier?

**Comment by kyessenov** [comment]
It's easier with a copy I think. If you just want a mention on the docs there, we're happy to link to your repo.

**Comment by antonengelhardt** [comment]
What do you think? https://github.com/istio-ecosystem/wasm-extensions/pull/95

**Comment by antonengelhardt** [comment]
@kyessenov Friendly ping
